codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/generators/ServerOperationHandlerGenerator.kt

```diff
@@ -157,6 +181,7 @@ class ServerOperationHandlerGenerator(
             Fut: std::future::Future<Output = $outputType> + Send,
             B: $serverCrate::body::HttpBody + Send + 'static, $streamingBodyTraitBounds
             B::Data: Send,
+            B: std::fmt::Debug,
```

Comment:
I don't really like adding debug bounds like this since sometimes it can be painful. I think we should avoid maybe logging bodies?

Reply:
Unless its an error then I would try to convert the body into a string to print that maybe?

Reply:
I was thinking of logging them at `TRACE` level, but they can contain `sensitive` data (https://awslabs.github.io/smithy/1.0/spec/core/documentation-traits.html#sensitive-trait), so it's best that we don't log anything by default. Perhaps we can do it in a separate Tower middleware users can opt into if they absolutely must log bodies.